deploy/example_clusters/cr-cluster-spark221e2-stor.yaml

```diff
@@ -1,37 +0,0 @@
-apiVersion: "kubedirector.hpe.com/v1beta1"
-kind: "KubeDirectorCluster"
```

Comment:
why are we removing this? We need to remove this from Everest also

Reply:
yes. i think there is a PR in everest for removing this. This image is buggy and we have newer spark 2.4.5 available in KD.